specs/pipelines/fail.spec

```diff
@@ -5,25 +5,38 @@ Contains negative scenarios that exercises running pipeline
 Precondition:
   * Operator should be installed
 
-## Run Task with a non-existent ServiceAccount
-Tags: e2e, tasks, negative
+## Run Pipeline with a non-existent ServiceAccount
+Tags: e2e, pipeline, negative
 
-Running a task using a ServiceAccount that does not exist must fail
+Running a pipeline using a ServiceAccount that does not exist must fail
 
 Steps:
-  * Create task
   * Verify ServiceAccount "foobar" does not exist
-  * Run task using "foobar" ServiceAccount
-  * Verify taskrun has failed
-
+  * Create
```

Comment:
`create resource from table` will be descriptive

Reply:
Agree

Reply:
I will update it on next pr